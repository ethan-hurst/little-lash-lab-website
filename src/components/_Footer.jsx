import React from 'react';

export default class _Footer extends React.Component {
    render() {
        return (
            <div className="Footer">
                <footer class="py-5 bg-black">
                    <div class="container">
                        <p class="m-0 text-center text-white small">Copyright &copy; The Little Lash Lab 2018   </p>
                    </div>
                </footer>
            </div>
        );
    }
}