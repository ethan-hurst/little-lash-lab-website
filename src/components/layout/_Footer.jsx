import React from 'react';
import InstagramFeed from '../InstagramFeed'

export default class _Footer extends React.Component {
  render() {
    return (
      <>
      <InstagramFeed count="8" />
      <div className="Footer">
        <footer className="py-5 bg-black">
          <div className="container">
            <p className="m-0 text-center text-white small">
              Copyright &copy; The Little Lash Lab
              {' '}
              {new Date().getFullYear()}
            </p>
          </div>
        </footer>
      </div>
      </>
    );
  }
}
