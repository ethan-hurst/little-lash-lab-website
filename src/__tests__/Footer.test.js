import { shallow } from '../enzyme';
import React from 'react';
import { create } from "react-test-renderer";
import _Footer from '../components/layout/_Footer';

test('should match the snapshot', () => {
    const c = create(<_Footer />)
    expect(c.toJSON()).toMatchSnapshot();
});