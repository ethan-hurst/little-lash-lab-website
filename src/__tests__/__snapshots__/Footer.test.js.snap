// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<div
  className="Footer"
>
  <footer
    class="py-5 bg-black"
  >
    <div
      class="container"
    >
      <p
        class="m-0 text-center text-white small"
      >
        Copyright © The Little Lash Lab 2019
      </p>
      <p
        class="m-0 text-center text-white small"
      >
        Copyright © Hursty By Design Ltd 2019
      </p>
    </div>
  </footer>
</div>
`;
